PythonAPI/examples/multi_chrono.py

```diff
@@ -0,0 +1,32 @@
+import carla
```

Comment:
@sina-sixwheel clarification about how this file is supposed to work:

- first run the manual_chrono_control.py script as normal to spawn the PyGame
- then, run this script to spawn the additional CarlaActors

is that right?